sale_invoice_auto_deliver/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Sale Invoice Automatic Deliver
+==============================
+
+The goal of this module is to automatically deliver products when an user create an invoice.
+On Sale Order when an user choose to create an invoice, he can use the option : "Invoice the whole sales order + Automatic Deliver"
+in order to deliver products at the same time than the invoice. This functionality can be useful in the case where products are already take out of the stock.
+
+When invoice sale order line it is possible to choose the option : "Create Invoice + Automatic Deliver" to deliver the product and invoice it.
+
+Consider as pre-requirement that all products must be available.
+
+Installation
+============
+
+To install this module, you need to:
+
+* Click on install button
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to ...
```

Comment:
There are no usage instructions. Remove this section if empty or move part of the main description that are usage instructions.